src/key.rs

```diff
@@ -251,6 +251,7 @@ async fn generate_keypair(context: &Context) -> Result<KeyPair> {
             let keytype = KeyGenType::from_i32(context.get_config_int(Config::KeyGenType).await?)
                 .unwrap_or_default();
             info!(context, "Generating keypair with type {}", keytype);
+            // TODO? add it here as well?
```

Comment:
I think so, at least for uniformity

Reply:
the question is how?
- Why is it using `Handle::current().spawn_blocking` instead of `tokio::task::spawn_blocking`?
- can I just replace it by `tokio::task::spawn_blocking` or do I need to make a new variant of the macro that takes `Handle::current()` into account?

Reply:
As far as i can tell from the Tokio code, `tokio::task::spawn_blocking()` is just a shortcut for `Handle::current().spawn_blocking()`:
https://github.com/tokio-rs/tokio/blob/29cd6ec1ec6f90a7ee1ad641c03e0e00badbcb0e/tokio/src/task/blocking.rs#L202
https://github.com/tokio-rs/tokio/blob/29cd6ec1ec6f90a7ee1ad641c03e0e00badbcb0e/tokio/src/runtime/blocking/pool.rs#L178
So `spawn_named_blocking_task!()` fits here as well.